config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-ipam
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+    - name: ANSIBLE_ENABLE_TASK_DEBUGGER
+      value: "True"
+  services:
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - ovn
+    - libvirt
+    - nova
+  deployStrategy:
+      deploy: true
+  baremetalSetTemplate:
+    deploymentSSHSecret: dataplane-ansible-ssh-private-key-secret
+    bmhLabelSelector:
+      app: openstack
+    ctlplaneInterface: enp1s0
+    dnsSearchDomains:
+      - osptest.openstack.org
+    cloudUserName: root
+  nodeTemplate:
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+    nodes:
+       edpm-compute-0:
+         hostName: edpm-compute-0
+         ansible:
+           ansibleUser: root
+           ansibleHost: 192.168.122.100
+           ansibleVars:
```

Comment:
These ansible vars would all be ignored when you use IPAM and specify the networks in nodeTemplate.